prow/config/config.go

```diff
@@ -53,6 +53,9 @@ const (
 	DefaultJobTimeout = 24 * time.Hour
 )
 
+// TODO: require setting this explicitly
+const defaultConfigPath = "/etc/config/config.yaml"
```

Comment:
Looks like this is only used by ConfigPath? If so you can scope the const inside the function.